src/components/TagFilterForm/TagFilterForm.js

```diff
@@ -120,7 +124,7 @@ class TagFilterForm extends Component {
               <TagContainer
                 id={tag}
                 showDelete
-                onDeleteClicked={() => {removeTag(tag); this.props.setFlipMove(false);}}
+                onDeleteClicked={() => this.removeTagFilter(tag)}
```

Comment:
Not exactly what I had in mind. `removeTagFilter` should be a function inside a function - two sets of parentheses - and then you just need to call it like `onDeleteClicked={this.removeTag(tag)}`. Let me know if you’d like info on curried functions.

Reply:
I meant `{this.removeTagFilter(tag)}`